src/core/server/rendering/views/template.tsx

```diff
@@ -164,7 +186,8 @@ export const Template: FunctionComponent<Props> = ({
         </div>
       );
     }
-    return openSearchLogoSpinner;
+    // Wazuh: change loading logo
+    return darkMode ? openSearchLogoSpinnerDark : openSearchLogoSpinner;
```

Comment:
I think this is causing an inconsistent and unexpected behavior while using the dark mode. Needs to be checked.